docs/UserGuide.md

```diff
@@ -192,17 +179,15 @@ Format: `delete MATRICULATION NUMBER`
   no student with the specified matriculation number can be found in the records.
 * If the student to be deleted has an appointment, the student's appointment will be deleted as well.
   
-
 Examples:
-* `delete A1234567X` deletes John Doe from the records. If John Doe has an appointment, his appointment will be deleted as well.
 
-| Before        | 
-| ------------- |
-|![deleteBefore](images/deleteBefore.png)|
+1. Suppose, you want to delete John Doe from the records. You can enter `delete A1234567X` and his corresponding appointment will be deleted as well.
```

Comment:
Because the delete command comes before addAppt, I don't think John Doe will have an appointment at this point. Best to say 'if he has an appointment then it will be deleted'